index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Home</title>
+    <link rel="stylesheet" href="https://element-ui.netlify.app/styles/utility.css">
+    <link rel="stylesheet" href="./styles/main.css">
+</head>
+
+<body>
+    <nav class="nav-container flex-row">
+        <a href="../index.html">
+            <h5>ezBuy</h5>
+        </a>
+        <input name="name" type="text" class="input-area text-base " placeholder="Search...">
+        <ul class="flex-row">
+            <li>
+                <a href="">
```

Comment:
why empty link

Reply:
acutally these are login,account,cart and wishlist link, will add them in the end.